ironfish/src/rpc/routes/wallet/postTransaction.test.ts

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import { v4 as uuid } from 'uuid'
+import { createRouteTest } from '../../../testUtilities/routeTest'
+
+describe('Route wallet/postTransaction', () => {
+  const routeTest = createRouteTest(true)
+
+  // TODO remove this when it's not needed for copying anymore
+  it('should return account status information', async () => {
+    const account = await routeTest.node.wallet.createAccount(uuid(), true)
+    const response = await routeTest.client
+      .request<any>('wallet/getAccountsStatus', {})
+      .waitForEnd()
+
+    expect(response.status).toBe(200)
+    expect(response.content).toMatchObject({
+      accounts: [
+        {
+          name: account.name,
+          id: account.id,
+          headHash: 'NULL',
+          headInChain: false,
+          sequence: 'NULL',
+        },
+      ],
+    })
+  })
+
+  // TODO a valid raw transaction
+  it('should return the posted transaction', async () => {
```

Comment:
You can use this, https://github.com/iron-fish/ironfish/blob/staging/ironfish/src/testUtilities/helpers/transaction.ts#L28-L64